### DISCUSSION TITLE
Don't let support of PlatformIO die.

### DISCUSSION THREAD

**Discussion by peteDDD** [espressif/arduino-esp32#8985] [Polls]
Please refer to the discussion here: https://github.com/espressif/arduino-esp32/pull/8606#issuecomment-1849949173

Please indicate if having ongoing, complete support of PlatformIO for new ESP-IDF releases is important to you.

**Comment by me-no-dev** [comment]
This is the Arduino repository, not the ESP-IDF one. Please write something about Arduino or open a new one in the relevant repository.

**Comment by lucasssvaz** [comment]
Just as an additional note, we are not responsible for PlatformIO support.

**Comment by SuGlider** [comment]
@peteDDD - Do you mean having PlatformIO + Arduino Core 3.0.0+ using IDF 5.1?
This is done already. @Jason2866 has the files and the platformio.ini settings.

Just follow the instructions in https://github.com/Jason2866/platform-espressif32#readme

**Comment by Jason2866** [comment]
Actual ESP IDF 5.1.2 is supported from Platformio in actual development branch (since a few days)
https://github.com/platformio/platform-espressif32

Can be used with
```
platform = https://github.com/platformio/platform-espressif32.git
```
